contracts/src/v0.8/automation/dev/v2_3/AutomationRegistryBase2_3.sol

```diff
@@ -975,29 +979,43 @@ abstract contract AutomationRegistryBase2_3 is ConfirmedOwner {
 
     PaymentReceipt memory receipt = _calculatePaymentAmount(hotVars, paymentParams);
```

Comment:
in the `PaymentReceipt` let's say:
* `gasCharge` / `premium` are in the billing token, using the billing token's native decimals
* `gasReimbursementJuels` / `premiumJuels` use LINK, which _always_ uses 18 decimals